mlrun/serving/routers.py

```diff
@@ -347,12 +443,12 @@ def _parallel_run(self, event: dict):
         for route in self.routes.keys():
             if self.executor_type == ParallelRunnerModes.process:
                 future = executor.submit(
-                    _ParallelRunInterface._wrap_step, route, copy.copy(event)
+                    ParallelRun._wrap_step, route, id(self), copy.copy(event)
```

Comment:
why `id(self)`? why use the `id`?

Reply:
We using it because `local_routes` is global and we want that each object will have is own space in this dictionary, to prevent override by different object that have routes with the same names.